python/mujinwebstackclient/controllerwebclientraw.py

```diff
@@ -51,13 +65,39 @@ def _SetJSONWebToken(self, response, *args, **kwargs):
         # switch to JWT authentication
         self._jsonWebToken = response.cookies.get('jwttoken')
 
+    def _GetAuthorization(self) -> str:
```

Comment:
This function is not a private function, it is being used outside of the `JSONWebTokenAuth` class

```suggestion
    def GetAuthorization(self) -> str:
```

Reply:
Renamed `GetAuthorization` to `GetAuthorizationHeader`.